<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1MD18DL" failures="2" tests="8" name="Test" time="50.943" errors="0" timestamp="2025-08-13T17:38:20 IST">
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="2.167"/>
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="4.181"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="2.359"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="2.113">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDataDrivenTest.verify_loginDDT(TC003_LoginDataDrivenTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="1.991">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDataDrivenTest.verify_loginDDT(TC003_LoginDataDrivenTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="11.816"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="1.952"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="verify_loginDDT" time="11.203"/>
</testsuite> <!-- Test -->
